lib/discordgo/endpoints.go

```diff
@@ -134,6 +134,9 @@ var (
 	EndpointChannelWebhooks = func(cID int64) string { return "" }
 	EndpointWebhook         = func(wID int64) string { return "" }
 	EndpointWebhookToken    = func(wID int64, token string) string { return "" }
+	EndpointWebhookMessage  = func(wID int64, token string, messageID int64) string {
```

Comment:
This here does not follow the logic YAG's endpoints.go is built up
this var section should be `EndpointWebhookMessage  = func(wID int64, token, messageID string) string { return "" }`

and you give the full inside CreateEndpoints function down below...

notice I use string for messageID type, why? it has `@original` possible > https://discord.com/developers/docs/interactions/receiving-and-responding#get-original-interaction-response

Reply:
There is currently a separate endpoint for original interaction response, if I add capability for @original to be used in WebhookMessage endpoint, shall I also remove the original interaction response endpoint? That would match upstream discordgo more closely iirc.

Reply:
closer to the upstream the better, less changes in the future

Reply:
Should there be a constant for `"@original"` to be passed to (s *Session) WebhookMessage? I can't find one on discordgo upstream, but I don't love needing to pass the same string literal for each use of the function.

Reply:
Ah, I think I see what you're saying now, that makes sense to me. Thanks so much for your wisdom and patience as always :)